sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/metrics/_exporter.py

```diff
@@ -148,14 +148,13 @@ def _convert_point_to_envelope(
         value = point.value
     elif isinstance(point, HistogramDataPoint):
         value = point.sum
-        count = point.count
+        count = int(point.count)
         min_ = point.min
         max_ = point.max
 
     data_point = MetricDataPoint(
         name=str(name)[:1024],
         value=value,
-        data_point_type="Aggregation",
```

Comment:
What's the purpose of this removal?

Reply:
Aligning with this [spec](https://github.com/microsoft/Telemetry-Collection-Spec/blob/main/OpenTelemetry/metrics/OpenTelemetryToAppInsightsMetricMapping.md#specification) for app insights mapping. SDKs shouldn't pass in `data_point_type`.